docs/explanation/e-interfaces.md

```diff
@@ -0,0 +1,28 @@
+# Interfaces/Endpoints
```

Comment:
Similar comment to the `config` one, there is an `integrations` tab in charmhub that is automatically generated.

Reply:
I will remove it.

Reply:
Done